### DISCUSSION TITLE
Debugging osx app

### DISCUSSION THREAD

**Discussion by alyfreym** [NationalSecurityAgency/ghidra#6341] [Q&A]
Hi, is it possible to debug osx  app (arm) using Ghidra?

**Comment by d-millar** [comment]
@alyfreym Yes, but the set-up work is a bit of a headache.  If you look in Ghidra/Debug/Debugger-swig-lldb, you'll find a script which MAY help you get set up automatically (fingers crossed) and the full set of instructions for what to do if it doesn't.  That said, feel free to ping us here if anything isn't clear.
